docs/src/main/sphinx/connector/query-passthrough-warning.fragment

```diff
@@ -0,0 +1,11 @@
+The native query passed to the underlying data source is required to return a
+table as a result set. This can be a one column table with one row. No
+validation or security checks of these queries are performed by Trino. The
+data source performs these tasks using its own configuration.
+
+You can use passthrough queries to read data; however, queries that write data
+typically do not return a table. As a result, when a write operation is passed,
+Trino declares that the query failed even if the database performs the
+operation. If supported by the underlying data source, you can avoid this
+outcome by appending a statement with ``RETURNING 1``. Performing this action
+may grant access to restricted data or result in data loss.
```

Comment:
I would remove "may grant access to restricted data". If you mean access which would be restricted if the query was run in Trino, then it is equally true for read and write operations.

Regarding "result in data loss", I would reword it. I a user runs a DELETE query, it is their expectation that data will be deleted. "Data loss" sounds like a bug.